src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -102,7 +102,7 @@ resource identityRoleAssignments 'Microsoft.Authorization/roleAssignments@2022-0
 // Stop hub triggers if they're already running
 resource stopHubTriggers 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
   name: '${dataFactoryName}_stopHubTriggers'
-  location: location
+  location: startsWith(location, 'china') ? 'chinaeast2' : location // Pin to the only region(chinaeast2) that supports deployment scripts in China
```

nit: Can we put this on the line above for readability?
```suggestion
  // chinaeast2 is the only region in China that supports deployment scripts
  location: startsWith(location, 'china') ? 'chinaeast2' : location
```